examples/getstarted/config/environments/development/server.json

```diff
@@ -1,6 +1,6 @@
 {
-  "host": "localhost",
-  "port": 1337,
+  "host": "${process.env.HOST || '0.0.0.0'}",
```

Comment:
That's right?

<img width="565" alt="Screenshot 2020-03-31 at 15 39 24" src="https://user-images.githubusercontent.com/1007051/78032802-d9220d00-7365-11ea-88fa-e2f297bb8eec.png">

Reply:
@abdonrd  what is your question ?

Reply:
Use the `0.0.0.0` in the development env. Previously show `localhost`.

Reply:
That's what I wanted indeed. the url isn't cool to read that's for sure :/ 
 but it's the only way to make it work with docker for example.